combirepo/dependency_graph_builder.py

```diff
@@ -324,10 +325,20 @@ def _search_dependencies(yum_sack, package, providers, preferables, strategy):
                 element = Set([requirement_name])
                 provided_symbols = provided_symbols | element
                 if len(provider) != 1:
-                    provider = _handle_have_choice_problem(requirement_name,
-                                                           provider,
-                                                           preferables,
-                                                           strategy)
+                    providers_in_list = 0
```

Comment:
Not a good name for counter. Maybe `num_providers` or `providers_count` or smth. else?

Reply:
Done